libpod/networking_linux.go

```diff
@@ -100,6 +101,9 @@ func (r *Runtime) configureNetNS(ctr *Container, ctrNS ns.NetNS) ([]*cnitypes.Re
 
 // Create and configure a new network namespace for a container
 func (r *Runtime) createNetNS(ctr *Container) (n ns.NetNS, q []*cnitypes.Result, err error) {
+	if rootless.IsRootless() {
+		return nil, nil, errors.New("cannot configure a new network namespace in rootless mode, only --mode=slirp4netns is supported")
```

Comment:
I'm not seeing or recalling a `--mode` option.  Did you mean `--network` or do I need to refill my tea cup?

Reply:
no my mistake, thanks for finding it.  Pushed a fixed version